<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pankkiSivu</class>
 <widget class="QMainWindow" name="pankkiSivu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>631</width>
          <height>351</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="1">
            <widget class="QLabel" name="SALDO_LB">
             <property name="font">
              <font>
               <family>Source Code Pro</family>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>SALDO</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="TAKAISIN_LB">
             <property name="font">
              <font>
               <family>Source Code Pro</family>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>TAKAISIN</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="LOPETA_LB">
             <property name="font">
              <font>
               <family>Source Code Pro</family>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>LOPETA</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="LOPETA_LB_2">
             <property name="font">
              <font>
               <family>Webdings</family>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="TAKAISIN_BT">
             <property name="font">
              <font>
               <family>Webdings</family>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="SALDO_BT">
             <property name="font">
              <font>
               <family>Webdings</family>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="OTTO_LB">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Source Code Pro</family>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>          OTTO</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="OTTO_BT">
             <property name="font">
              <font>
               <family>Webdings</family>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>4</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="TALLETUS_LB">
             <property name="font">
              <font>
               <family>Source Code Pro</family>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string>      TALLETUS</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="TALLETUS_BT">
             <property name="font">
              <font>
               <family>Webdings</family>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>4</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="TILITAPAHTUMAT_LB">
             <property name="font">
              <font>
               <family>Source Code Pro</family>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>ArrowCursor</cursorShape>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>TILITAPAHTUMAT</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="TILITAPAHTUMAT_BT">
             <property name="font">
              <font>
               <family>Webdings</family>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>4</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPANKKISIVU">
    <property name="title">
     <string>PANKKISIVU</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuYES">
    <property name="title">
     <string>YES</string>
    </property>
   </widget>
   <addaction name="menuPANKKISIVU"/>
   <addaction name="menuYES"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
